src/modules/init/index.ts

```diff
@@ -73,10 +114,10 @@ export default async () => {
   log.debug('Prompting for app info')
   log.info('Hello! I will help you generate basic files and folders for your app.')
   try {
-    const repo = templates[await promptTemplates()]
+    const { repository: repo, organization: org } = templates[await promptTemplates()]
     await promptContinue(repo)
-    log.info(`Cloning https://vtex-apps/${repo}.git`)
-    await git.clone(repo)
+    log.info(`Cloning https://${org}/${repo}.git`)
```

Comment:
Could you remove the `https://` and the `.git`? I think it's non important info for the user, what do you think?